pkg/omdb/omdb.go

```diff
@@ -28,7 +29,7 @@ func New(apiKey string) Client {
 	}
 	return &Omdb{
 		ApiKey: apiKey,
-		Cache:  NewMemoryCache(),
```

Comment:
please use NewMemoryCache in case we add some specific initialization to the struct